### PR TITLE
Add build check alert

### PR DESCRIPTION
**Pull request by srikanthops**
add alerts when build fails > 24 hours

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -381,6 +451,29 @@ jobs:
...
+          branch: "master"
+          workflow: "dev_build_and_deploy"
+      - slack/status:
+          channel: cloudstate-alerts
+          failure_message: Builds failing on CircleCI for more then 1 day
+          fail_only: true
+          only_for_branches: "master"
+      - pagerduty/notify-on-failure:
+          from_account: "srikanth.koneru@lightbend.com"
+          incident_description: Builds are faling on circleCI
```
Comment (marcoderama) [inline .circleci/config.yml:469]:
```suggestion
          incident_description: Builds are failing on CircleCI
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -261,6 +261,76 @@ commands:
...
+            buildResp=$(curl -s -X GET -H 'Accept: application/json' -H "Circle-Token: ${CIRCLECI_TOKEN}" \
+              "https://circleci.com/api/v2/insights/${projectslug}/workflows/${workflow}?branch=${branch}" | \
+              jq '.items[0]')
+            if [[ -n "${buildResp}" ]]; then
+              jstatus=$(echo "${buildResp}" | jq .status|tr -d '"')
+              ctime=$(echo "${buildResp}" | jq .created_at|tr -d '"')
+              stime=$(echo "${buildResp}" | jq .stopped_at|tr -d '"')
+              if [[ "${jstatus}" == "success" ]]; then
+                #https://app.circleci.com/pipelines/github/lightbend/cloudstatemachine/3857/workflows/5d522640-8b04-4980-8a4e-11337ac4ffcd
+                echo "had successful build finished at:$stime"
```
Comment (marcoderama) [inline .circleci/config.yml:296]:
```suggestion
                echo "Successful build finished at:$stime"
```

File: .circleci/config.yml (hunk 1 above)
Comment (edwardcallahan) [reply .circleci/config.yml:469]: Falling sounds even worse!

File: .circleci/config.yml [hunk 3]
```diff
@@ -393,7 +486,11 @@ workflows:
 
   test:
     jobs:
-
+      - verify-build-check:
+        filters:
+          branches:
+            only:
+              - cron-test
```
Comment (edwardcallahan) [inline .circleci/config.yml:493]: we are still using `cron-test`? thought we moved off it.

**Pull request merged**